qiskit_addon_sqd/fermion.py

```diff
@@ -350,6 +355,53 @@ def bitstring_matrix_to_sorted_addresses(
     return addresses_left, addresses_right
 
 
+def bitstring_matrix_to_ci_strs(
+    bitstring_matrix: np.ndarray, open_shell: bool = False
+) -> tuple[np.ndarray, np.ndarray]:
+    """
+    Convert bitstrings (rows) in a ``bitstring_matrix`` into base-10 integer representations of determinants.
```

Comment:
They're just integers, not "base-10" integers.

Reply:
I started thinking that integers can be specified as base-2, which is a pretty natural way to specify these configurations as well. So I thought being explicit wouldn't hurt

Reply:
The integers _can_ be specified in base-2, or any other base. That's my point. The base 10 is not special here, nor in a general software context.

Reply:
Removed all base-10 mentions in fermion module

Reply:
Well, on further thought, base 2 is special here, because the integer representation of a determinant is obtained by converting its string representation to an integer, in base 2.